crates/bitwarden-generators/src/generator_client.rs

```diff
@@ -1,11 +1,14 @@
 use bitwarden_core::Client;
+#[cfg(feature = "wasm")]
+use wasm_bindgen::prelude::*;
 
 use crate::{
     passphrase::passphrase, password::password, username::username, PassphraseError,
     PassphraseGeneratorRequest, PasswordError, PasswordGeneratorRequest, UsernameError,
     UsernameGeneratorRequest,
 };
 
+#[cfg_attr(feature = "wasm", wasm_bindgen)]
 pub struct GeneratorClient {
     client: Client,
 }
```

Comment:
You also need to tag the `impl GeneratorClient {` line with `#[cfg_attr(feature = "wasm", wasm_bindgen)]` otherwise the functions in the client won't be available in the WASM build.


Like this: 
https://github.com/bitwarden/sdk-internal/pull/222/files#diff-809b50effbd071dc13d8b45a3e44fea9d265405ab5bfa4efd23e343334510443R16